broker/consumer/vm_startup.sh

```diff
@@ -87,17 +87,28 @@ do
     # which kills the while loop. no working suggestions found.
     # passing the error with `|| true`
 
+    
+    IFS=', ' read -r -a KAFKA_TOPIC_ARRAY <<< "$KAFKA_TOPIC"
```

Comment:
This should go in the section at the top of the file that defines topic-related variables. Actually there's currently two sections, starting on lines 49 and 52, I would combine them into one and add this there.

Add a comment describing what this line does. Shell is not as intuitive to read as python, so more comments are helpful.